core/src/data/value/binary_op/f32.rs

```diff
@@ -446,6 +446,11 @@ mod tests {
         assert!(
             matches!(base.try_divide(&Decimal(Decimal::ONE)), Ok(Decimal(x)) if x == Decimal::ONE)
         );
+        // Test with Decimal::TWO to ensure division (not multiplication) is performed
+        let base_two = 2.0_f32;
+        assert!(
+            matches!(base_two.try_divide(&Decimal(Decimal::TWO)), Ok(Decimal(x)) if x == Decimal::ONE)
+        );
```

Comment:
we don't need to have two unit tests for `try_divide`. having only one would be enough.
Let's remove the above one. we can replace to the added one.
then we can remove declaring new variable with `base_two` name, too.
and this test is already simple enough to understand so having comment would be too much.

we can apply same stuff to f64